schemas/undernames.js

```diff
@@ -0,0 +1,24 @@
+const increaseUndernamesSchema = {
+  $id: '#/definitions/increaseUndernames',
+  type: 'object',
+  properties: {
+    function: {
+      type: 'string',
+      const: 'increaseUndernames',
```

Comment:
wondering if we should call this `increaseUndernameCount` or `incrementUndernameCount` - as we're not adding actual `undernames` just the ability to serve more from an ANT

Reply:
added